sql/2020/09_Performance/web_vitals_by_device.sql

```diff
@@ -0,0 +1,153 @@
+#standardSQL
+# Core WebVitals by device
+
+CREATE TEMP FUNCTION IS_GOOD (good FLOAT64, needs_improvement FLOAT64, poor FLOAT64) RETURNS BOOL AS (
+  good / (good + needs_improvement + poor) >= 0.75
+);
+
+CREATE TEMP FUNCTION IS_NI (good FLOAT64, needs_improvement FLOAT64, poor FLOAT64) RETURNS BOOL AS (
+  good / (good + needs_improvement + poor) < 0.75
+  AND poor / (good + needs_improvement + poor) < 0.25
+);
+
+CREATE TEMP FUNCTION IS_POOR (good FLOAT64, needs_improvement FLOAT64, poor FLOAT64) RETURNS BOOL AS (
+  poor / (good + needs_improvement + poor) >= 0.25
+);
+
+CREATE TEMP FUNCTION IS_NON_ZERO (good FLOAT64, needs_improvement FLOAT64, poor FLOAT64) RETURNS BOOL AS (
+  good + needs_improvement + poor > 0
+);
+
+WITH
+  base AS (
+  SELECT
+    origin,
+    device,
+
+    fast_fid,
+    avg_fid,
+    slow_fid,
+    
+    fast_lcp,
+    avg_lcp,
+    slow_lcp,
+    
+    small_cls,
+    medium_cls,
+    large_cls,
+
+    fast_fcp,
+    avg_fcp,
+    slow_fcp,
+
+    fast_ttfb,
+    avg_ttfb,
+    slow_ttfb
+
+  FROM
+    `chrome-ux-report.materialized.device_summary`
+  WHERE
+    device IN ('desktop','phone')
+    AND date >= '2020-09-01'
+    AND date <= "2020-09-30"
```

Comment:
No need to cover a range of dates. And this should reference the August dataset.
```suggestion
    AND date >= '2020-08-01'
```